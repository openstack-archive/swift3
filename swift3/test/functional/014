#!/bin/bash

# Abort Multipart Upload

. ./common

num=10

S3USER=tester


_s3_put /bucket

upload_id=$(_s3_post /bucket/object?uploads | \
    _xq /InitiateMultipartUploadResult/UploadId)

for i in `seq $num`; do
    j=$(($i - 1))
    _s3_put /bucket/object?uploadId=${upload_id}\&partNumber=$i \
        -T ./data/segment_1M.dat_0$j
done

_s3_delete /bucket/object?uploadId=${upload_id}

