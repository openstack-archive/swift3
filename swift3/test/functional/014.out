QA output created by 014
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X POST http://SWIFT_HOST/bucket/object?uploads... 200
> s3curl --id tester -- -X PUT -T ./segment_1M.dat_00 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=1... 200
> s3curl --id tester -- -X PUT -T ./segment_1M.dat_01 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=2... 200
> s3curl --id tester -- -X PUT -T ./segment_1M.dat_02 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=3... 200
> s3curl --id tester -- -X PUT -T ./segment_1M.dat_03 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=4... 200
> s3curl --id tester -- -X PUT -T ./segment_1M.dat_04 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=5... 200
> s3curl --id tester -- -X PUT -T ./segment_1M.dat_05 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=6... 200
> s3curl --id tester -- -X PUT -T ./segment_1M.dat_06 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=7... 200
> s3curl --id tester -- -X PUT -T ./segment_1M.dat_07 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=8... 200
> s3curl --id tester -- -X PUT -T ./segment_1M.dat_08 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=9... 200
> s3curl --id tester -- -X PUT -T ./segment_1M.dat_09 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=10... 200
> s3curl --id tester -- -X DELETE http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID... 204
